src/components/UserInfo/UserName.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import UserInfoModel from '../../models/userInfoModel';
+import {StyledUserName} from './styles';
+
+interface UserNameProps {
+  getUserInfo: () => UserInfoModel | null;
+}
+
+const UserName: React.SFC<UserNameProps> = ({getUserInfo}) =>
+  getUserInfo() ? (
```

Comment:
May be it makes sense to store the result of the method into variable. if it is true you call it twice